### PR TITLE
Add new filetypes

### PR DESCRIPTION
**Pull request by yochem**
- project files `CHANGELOG{,.md,.txt}`, `CODE_OF_CONDUCT{,.md,.txt}`, `CONTRIBUTING{,.md,.txt}`
- Vlang files [`.v`, `v.mod`] (https://github.com/vlang/v)
- Golang files [`go.mod`, `go.sum`] (https://golangbyexample.com/go-mod-sum-module/)
- CI: `.cirrus.yml` (https://cirrus-ci.org/)
- Mac: `Icon\r` (Icon files for Finder, `\r` is necessary for it to work)
- editorconfig: `.editorconfig` (https://editorconfig.org/)
- other: `.ctags` (not very important, since they're always in a `.ctags.d/` dir)

### CODE REVIEW

File: config/filetypes.yml [hunk 1]
```diff
@@ -408,15 +428,18 @@ unimportant:
     - .CFUserTextEncoding
     - .DS_Store
     - .localized
+    - Icon\r
```
Comment (sharkdp) [inline config/filetypes.yml:431]: I think this is a mistake?

Comment (sharkdp) [reply config/filetypes.yml:431]: Oh, you said it's needed in the description. Why?

Comment (yochem) [reply config/filetypes.yml:431]: I don't know exactly why the filename contains a carriage return, but some more information can be found here: https://superuser.com/a/298798.

Comment (sharkdp) [reply config/filetypes.yml:431]: And can you confirm that this works properly with `vivid` + GNU ls?

Comment (yochem) [reply config/filetypes.yml:431]: ~/.config/vivid/filetypes.yml:
```yaml
test:
  - Icon\r
```
~/.config/vivid/one-dark-simple.yml:
```yaml
...
test:
  foreground: red
```

image

Comment (yochem) [reply config/filetypes.yml:431]: Tested on MacOS, gls v9.3, fish

Comment (sharkdp) [reply config/filetypes.yml:431]: Thank you

**Review by sharkdp — APPROVED** [review]

**Pull request merged**